From d5d7ecf0fd27e95dc3dd0ae5c1670e4da2249f50 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Sun, 19 Oct 2025 12:12:43 +0300
Subject: [PATCH 496/553] util/{unicode,xkbdatagen}: fix build with
 `REUSE-IgnoreStart`

Building util/unicode I get:
main.cpp:3368:7: error: invalid preprocessing directive #REUSE
 3368 |     # REUSE-IgnoreStart
      |       ^~~~~
main.cpp:3374:7: error: invalid preprocessing directive #REUSE
 3374 |     # REUSE-IgnoreEnd
      |       ^~~~~

Amends a53be8b935964836b643f1b40d56fe16a35cf9c4.

Pick-to: 6.8
Change-Id: I21c5c6ecc0a8095d74c3721acf77274a1cc78d60
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
Reviewed-by: Marc Mutz <marc.mutz@qt.io>
(cherry picked from commit 14f0f48487e5a8797ef412887ffa357d6c48aaae)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 util/unicode/main.cpp    | 4 ++--
 util/xkbdatagen/main.cpp | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/util/unicode/main.cpp b/util/unicode/main.cpp
index 1f31febeaaf..d303a369812 100644
--- a/util/unicode/main.cpp
+++ b/util/unicode/main.cpp
@@ -3365,13 +3365,13 @@ int main(int, char **)
     QByteArray normalizationCorrections = createNormalizationCorrections();
     QByteArray idnaMapping = createIdnaMapping();
 
-    # REUSE-IgnoreStart
+    // REUSE-IgnoreStart
     QByteArray header =
         "// Copyright (C) 2020 The Qt Company Ltd.\n"
         "// SPDX-License-Identifier: Unicode-3.0\n"
         "// Qt-Security score:significant reason:default\n"
         "\n";
-    # REUSE-IgnoreEnd
+    // REUSE-IgnoreEnd
 
     QByteArray note =
         "/* This file is autogenerated from the Unicode " DATA_VERSION_S " database. Do not edit */\n\n";
diff --git a/util/xkbdatagen/main.cpp b/util/xkbdatagen/main.cpp
index 4c9227816b1..df0a1be7dbd 100644
--- a/util/xkbdatagen/main.cpp
+++ b/util/xkbdatagen/main.cpp
@@ -374,14 +374,14 @@ int main(int argc, char **argv)
     }
 
     QList<XKBLayout> layouts = findLayouts(layoutList);
-    # REUSE-IgnoreStart
+    // REUSE-IgnoreStart
     // copyright and stuff
     printf("// Copyright (C) 2016 The Qt Company Ltd.\n"
            "// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only\n"
            "// This file is auto-generated, do not edit!\n"
            "// (Generated using util/xkbdatagen)\n"
            "\n");
-    # REUSE-IgnoreEnd
+    // REUSE-IgnoreEnd
 
     // data structure
     printf("static struct {\n"
-- 
2.51.2

